### DISCUSSION TITLE
vertical (min-height) breakpoints

### DISCUSSION THREAD

**Discussion by aroman** [chakra-ui/chakra-ui#8915] [Ideas]
### Description

allow specification of breakpoints as vertical instead of horizontal

### Problem Statement/Justification

we're building a mobile-only webapp. horizontal breakpoints are much less useful to use than vertical breakpoints, as phones can vary quite a bit in vertical resolution even at the same horizontal value

### Proposed Solution or API

```ts
breakpoints: {
    options: { vertical: true }
    base: '0px',
    xs: '1px',
    sm: '380px',
    md: '768px',
    lg: '992px',
    xl: '1280px'
    '2xl': '1536px'
  },
  ```

**Comment by jwj777** [comment]
This would be nice
